src/coreclr/gc/gc.cpp

```diff
@@ -29626,17 +30018,20 @@ void gc_heap::pin_object (uint8_t* o, uint8_t** ppObject)
 
 size_t gc_heap::get_total_pinned_objects()
 {
-#ifdef MULTIPLE_HEAPS
-    size_t total_num_pinned_objects = 0;
-    for (int i = 0; i < gc_heap::n_heaps; i++)
-    {
-        gc_heap* hp = gc_heap::g_heaps[i];
-        total_num_pinned_objects += hp->num_pinned_objects;
-    }
-    return total_num_pinned_objects;
-#else //MULTIPLE_HEAPS
-    return num_pinned_objects;
-#endif //MULTIPLE_HEAPS
+    //TEMP BEG
+    return n_heaps;
```

Comment:
Is this intentional to return n_heap for the count of the pinned objects?

Reply:
feels like some test change which needs to be reverted.

Reply:
I will revert it (yes it was for testing). this is not in the final state yet. I just wanted to submit to make sure it doesn't have build issues on some OSs that I'm not building locally.